charts/open-notificaties/templates/configmap.yaml

```diff
@@ -24,6 +24,15 @@ data:
   {{- end }}
   IS_HTTPS: {{ if .Values.settings.isHttps }}"True"{{ else }}"False"{{ end }}
   RABBITMQ_HOST: {{ .Values.settings.messageBroker.host }}
+  {{- if not eq .Values.settings.autoRetry.maxRetries nil }}
+  NOTIFICATION_DELIVERY_MAX_RETRIES: {{ .Values.settings.autoRetry.maxRetries | toString | quote }}
+  {{- end }}
+  {{- if .Values.settings.autoRetry.backoff }}
+  NOTIFICATION_DELIVERY_RETRY_BACKOFF: {{ .Values.settings.autoRetry.backoff | toString | quote}}
```

Comment:
This one is tricky, in the values yaml this can be a boolean or a number (int or float) - but if it's a boolean, this needs proper casting to a string for environment variables. That should also happen for numbers actually, but ideally I'd like to convert boolean -> "1" or "0", and have numbers as "3"` for example.

Reply:
@sergei-maertens I don't understand why you want that conversion to happen? This would mean that a `True` in the values will get translated in the end to `1` as injected env-var in the container. Then in the actual code you would need to do another conversion back to `True`, as that is expected by the Celery Task property right? That would be quite hard because what if you would like `1` to be the actual value for `retry_backoff`? How would you differentiate those?

Keeping the code as is would work sufficient I would say. In the end it will translate to Container env-vars like this:

```
settings.autoRetry.maxRetries: 3 >>> NOTIFICATION_DELIVERY_RETRY_BACKOFF=3
settings.autoRetry.maxRetries: "True" >>> NOTIFICATION_DELIVERY_RETRY_BACKOFF=True
```

And that should work fine I guess. But perhaps I misunderstood you, so would be great if you could elaborate. 

PS: we're about to deploy v1.4.0 of O-N and would be great to also use the updated Helm chart (though not a hard requirement as the sane defaults for these settings are fine for now).

Reply:
The application casts the environment variable to an int. A string value of `True` or `true` cannot be cast to an int. This is because this parameter is actually a factor applied to the exponential backoff ([NOTIFICATION_DELIVERY_RETRY_BACKOFF](https://github.com/open-zaak/open-notificaties/blob/main/src/nrc/conf/includes/base.py#L474))

In python, `True` is equal to the int `1` as boolean is just a subclass of `int`, but what is actually required for celery is an int or a float. So, booleans sort of "work by accident".